compiler/bin-js_of_ocaml/compile.ml

```diff
@@ -65,6 +66,14 @@ let run
   | `Name _, _ -> ());
   List.iter params ~f:(fun (s, v) -> Config.Param.set s v);
   List.iter static_env ~f:(fun (s, v) -> Eval.set_static_env s v);
+  Config.Flag.(
+    match target_env with
+    | `Nodejs | `Isomorphic -> enable "include-node"
+    | `Browser -> disable "include-node");
+  Config.Flag.(
+    match target_env with
+    | `Browser | `Isomorphic -> enable "include-browser"
```

Comment:
Is this really what we want ? In this PR, you change the runtime with the following piece of code. I don't think we want it to be included in the isomorphic  case, do we ?
```
//Provides: fs_node_supported
//If: browser
function fs_node_supported () {
  return false
}
```

Reply:
It is correct, but it seems the way I expressed it wasn’t abundantly clear. This can be made more clear, so I’ll send a patch later to do so

Reply:
e43a7633d6

Reply:
e43a7633d6